receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -483,9 +499,228 @@ metrics:
+    unit: 1
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ cluster_state_update_type ]
+    enabled: true
+  elasticsearch.node.cluster.state_update.notification_time:
+    description: The cumulative amount of time spent notifying listeners of a cluster state update since the node started.
+    unit: 1
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ cluster_state_update_type ]
+    enabled: true
+  elasticsearch.node.ingest.count:
+    description: Total number of documents ingested during the lifetime of this node.
+    unit: "{documents}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.ingest.current:
+    description: Total number of documents currently being ingested.
+    unit: "{documents}"
+    gauge:
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.ingest.failed:
+    description: Total number of failed ingest operations during the lifetime of this node.
+    unit: "{operation}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.ingest.pipeline.count:
+    description: Total number of documents ingested during the lifetime of this node.
+    unit: "{documents}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ingest_pipeline_name ]
+    enabled: true
+  elasticsearch.node.ingest.pipeline.failed:
+    description: Total number of failed operations for the ingest pipeline.
+    unit: "{operation}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ingest_pipeline_name ]
+    enabled: true
+  elasticsearch.node.ingest.pipeline.current:
+    description: Total number of documents currently being ingested by a pipeline.
+    unit: "{documents}"
+    gauge:
+      value_type: int
+    attributes: [ ingest_pipeline_name ]
+    enabled: true
+  elasticsearch.node.script.compilations:
+    description: Total number of inline script compilations performed by the node.
+    unit: "{compilations}"
     sum:
       monotonic: false
       aggregation: cumulative
       value_type: int
-    attributes: [memory_state]
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.script.cache_evictions:
+    description: Total number of times the script cache has evicted old data.
+    unit: 1
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ]
+    enabled: true
+  elasticsearch.node.script.compilation_limit_triggered:
+    description: Total number of times the script compilation circuit breaker has limited inline script compilations.
+    unit: 1
+    sum:
+      monotonic: false
```

Comment:
Based on the description, this sounds monotonic. When would it decrease?

Reply:
Oh good point. I missed this one.